sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonPlugin.java

```diff
@@ -105,6 +107,12 @@ public void define(Context context) {
       PythonRuleRepository.class);
 
     if (context.getRuntime().getProduct() != SonarProduct.SONARLINT) {
+      if (context.getRuntime().getApiVersion().isGreaterThanOrEqual(SQ_MIN_VERSION_ANALYSIS_WARNINGS)) {
+        context.addExtension(DefaultAnalysisWarningsWrapper.class);
+      } else {
+        context.addExtension(NoOpAnalysisWarningsWrapper.class);
```

Comment:
Why not having the NoOpAnalysisWarningsWrapper also be included in SonarLint context? What about failing on something else than reports? (for instance during SquidSensor scan)

Reply:
I don't see the need to add `NoOpAnalysisWarningsWrapper`, I believe this is only displayed on SonarQube, and currently only sensors defined outside of SonarLint are raising UI warnings.

Indeed, other sensors can also crash the analysis. I will update implementation accordingly